confidence_edge_cases/github_context.py

```
@@ -0,0 +1,21 @@
+import hashlib
+import os
+
+
+def load_user_by_email(conn, email):
+    query = f"SELECT id, email, role FROM users WHERE email = '{email}'"
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)